src/modules/AieraChat/components/Messages/Prompt/index.tsx

```diff
@@ -121,17 +122,40 @@ export function Prompt({ onSubmit, onOpenSources, submitting, className }: Promp
                     Type your questions here...
                 </span>
             )}
+            <button
+                onClick={() => onToggleWebSearch()}
```

Comment:
Does this need an arrow function or could it just be `onClick={onToggleWebSearch}`?